base/reducedim.jl

```diff
@@ -1231,6 +1231,150 @@ end
 
 reducedim1(R, A) = length(axes1(R)) == 1
 
+function findextrema!(f, op_mn, op_mx, Rval_mn, Rind_mn, Rval_mx, Rind_mx, A::AbstractArray{T,N}) where {T,N}
```

Comment:
IIUC, this is not for public usage. Then we'd better add a `_` to mark this as an internal method.

Reply:
Correct. `_` prefix added.